blockchain/source/contracts/test/Dummy.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.23;
+import "../Administratable.sol";
+
+contract Dummy is Administratable {
```

Comment:
Somehow, mark this contract as test-enviroment part

Reply:
done